frontstage/__init__.py

```diff
@@ -10,6 +9,13 @@
                           db=app.config['REDIS_DB'])
 csrf = CSRFProtect(app)
 
+
+@app.context_processor
+def inject_availability_message():
+    if len(redis.keys('AVAILABILITY_MESSAGE')) == 1:
+        return dict(availability_message=redis.get('AVAILABILITY_MESSAGE').decode('utf-8'))
```

Comment:
Minor, but instead of dict() it's neater to use curly brackets

Reply:
Done.